src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+public class Task {
+    final private String name;
```

Comment:
Perhaps the modifiers could be in the [standard order](https://se-education.org/guides/conventions/java/index.html#methods)? Separately, you may also want to have a `setName` functionality in the future, so making `name` `final` may not be necessary.
```suggestion
    private final String name;
```